main.rb

```diff
@@ -1,47 +1,45 @@
 require './app'
 
 class Main
+  MENU_OPTIONS = {
+    1 => { message: 'List all books', method: :list_books },
+    2 => { message: 'List all people', method: :list_people },
+    3 => { message: 'Create a person', method: :create_person },
+    4 => { message: 'Create a book', method: :create_book },
+    5 => { message: 'Create a rental', method: :create_rental },
+    6 => { message: 'List rentals for a given person id', method: :list_rentals },
+    7 => { message: 'Quit', method: :quit }
```

Comment:
## OPTIONAL 
* I strongly suggest you fix this error i get when i try to quit the app 
![image](https://user-images.githubusercontent.com/52497246/228187993-b7479546-1409-437f-bce7-1fac3a05c2d8.png)